websockets/README.md

```diff
@@ -50,9 +50,27 @@ WebSockets have no built in authentication mechanism. Server-side processes list
 
 ## Encryption
 
-At the time of writing, the negotiated authentication mechanism should also be used to encrypt all traffic sent over the WebSocket even if TLS certificates are also used at the transport layer.
+Server-side processes listening on WebSocket addresses should use TLS certificates to secure transmitted data at the transport level.
 
-A mechanism to avoid this but also maintain backwards compatibility with existing server-side processes will be specified in a future revision to this spec.
+This does not provide any assurance that the remote peer possesses the private key that corresponds to their public key, so an additional handshake is necessary.
+
+During connection establishment over WebSockets, before the connection is made available to the rest of the application, if all of the following criteria are met:
+
+1. `noise` is negotiated as the connection encryption protocol
```

Comment:
So despite `noise` being negotiated as the encryption protocol, no encryption is done at the libp2p level, because of the `handshake_only` which overrules it?

Reply:
That's correct.  Noise does a handshake and encryption, and here we only want the handshake since encryption is done by TLS at the transport layer.  This is similar to how the WebTransport and WebRTC transports work.

We still negotiate a connection encryption protocol for backwards compatibility and to allow for alternatives to noise in the future.